include/swift/SILOptimizer/Differentiation/DifferentiationInvoker.h

```diff
@@ -71,8 +71,15 @@ struct DifferentiationInvoker {
 
     /// The parent `apply` instruction and the witness associated with the
     /// `IndirectDifferentiation` case.
-    std::pair<ApplyInst *, SILDifferentiabilityWitness *>
-        indirectDifferentiation;
+    ///
+    /// Note: This used to be a std::pair, but on FreeBSD std::pair can't be
```

Comment:
I believe, this is not FreeBSD specifics, but a libc++ one. Change the comment accordingly.

Reply:
FreeBSD specifically disables the copy constructor by setting `_LIBCPP_DEPRECATED_ABI_DISABLE_PAIR_TRIVIAL_COPY_CTOR` to preserve ABI. It is not a general limitation of libc++.